app/models/partners/child.rb

```diff
@@ -11,7 +11,8 @@
 #  first_name           :string
 #  gender               :string
 #  health_insurance     :jsonb
-#  item_needed_diaperid :integer
+#  item_needed_diaperid :integer          # DEPRECATED
+#  items_needed_ids     :string
```

Comment:
Why do we have this field if we have a join table?

Reply:
removed 3a43674610cfcffc3e402171a067d9b4da290af2